platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -985,6 +993,6 @@
 		.de("Ihr aktuelles Passwort ist zu alt.");
 	I18n0 YOUR_CURRENT_PASSWORD_MIGHT_HAVE_BEEN_COMPROMISED = new I18n0("Your current password might have been compromised.")//
 		.de("Ihr aktuelles Passwort wurde möglicherweise kompromittiert.");
-	I18n1 YOUR_LANGUAGE_HAS_BEEN_CHANGED_TO_ARG1 = new I18n1("Your language has been changed to '%s'.")//
-		.de("Ihre Sprache wurde geändert auf '%s'.");
+	I18n0 YOUR_LOCALIZATION_HAS_BEEN_CHANGED = new I18n0("Your localization has been changed.")//
+		.de("Ihre Lokalisation wurde geändert.");
```

Comment:
see above; we should settle for either `Lokalisation` or `Lokalisierung`

Reply:
done